packages/fhir-sanitize-param/README.md

```diff
@@ -74,8 +74,9 @@ Is this parameter required.
 Type: `String`  
 Required: `false`  
 
-### Valid Types
+**NOTE:** Arguments present in the request that have not been specified in the array of allowed arguments will be passed through un-sanitized under a separate key `args._raw`. You will need to perform your own validation on this afterwards.
```

Comment:
did we decide to still use _raw?

Reply:
Nope! This needs to be removed. I believe this was because the plan is to pass the whole request through, just in case they need/want to do custom stuff.

Reply:
Should I just remove it, or change it to a note explaining that core will pass through the whole request if they want to use params that we don't support?

Reply:
You can remove it, We'll note that in the Core wiki somewhere when we make the update